### ISSUE TITLE
Unable to exclude users from suggested reviewers list

### ISSUE THREAD

**Issue by yakirm-cr** [linear-b/gitstream#21]
**Describe the bug**

I defined under the `config.user_mapping` at `.cm/gitstream.cm` mapping for some of the users who are no longer defined as contributors for the repository with a value of `null` (as suggested [here](https://docs.gitstream.cm/cm-file/)).
Unfortunately, the `gitstream-cm bot` still lists those users in the suggested reviewers list as part of the PR comment. 

**To Reproduce**

1. Add the following section to the `.cm/gitstream.cm` in the default branch (on top the initial file provided as part of the instructions [here](https://docs.gitstream.cm/github-installation/)):  
   ```yaml
   config:
     user_mapping:
       - 'Invalid User <invaliduser@company.com>': 'InvalidUser'
       - 'InvalidUser <12345678+InvalidUser@users.noreply.github.com>': 'InvalidUser'
       - 'InvalidUser': null
       - 'Invalid User2 <invaliduser2@company.com>': 'InvalidUser2'
       - 'InvalidUser2': null
   ```
2. Open a PR for a new feature branch which includes at least one file modified by one of the users (or more) mentioned in step 1 above.
3. Required job `gitstream-cm` is executed for the PR successfully.

**Expected behavior**

Required job `gitstream-cm` is executed successfully and a comment is added to the PR with suggested reviewers that do NOT include the invalid users mapped to `null` value above.

** Actual behavior **

Required job `gitstream-cm` is executed successfully and a comment is added to the PR with the following content:

```md
 👋  **Suggested reviewers: ValidUser2**
 
They contributed more than 25% of the lines on pre-existing files:

`path/to/file1` 
ValidUser1: 44% 
ValidUser3: 18% 
ValidUser2: 12% 
ValidUser4: 11% 
ValidUser5: 3% 
ValidUser6: 3% 
ValidUser7: 2% 
ValidUser8: 1% 
ValidUser9: 1% 
ValidUser10: 0% 

`path/to/file2` 
ValidUser2: 45% 
ValidUser1: 15% 
ValidUser3: 10% 
ValidUser4: 7% 
ValidUser5: 7% 
ValidUser6: 4% 
InvalidUser: 3% 
ValidUser7: 2% 
InvalidUser2: 0% 
ValidUser8: 0% 
ValidUser9: 0% 
ValidUser10: 0% 
ValidUser11: 0% 

 
To learn more about /:\ gitStream - [Visit our Docs](https://docs.gitstream.cm/) 
 ```

*`InvalidUser` and `InvalidUser2` are listed as part of the contributors to `path/to/file2` (we expect them to be filtered out).*

**Additional context**

The repository on which `gitstream` was enabled is a private repository (so, the real data had to be redacted).
In addition, I tried to change the order of the entries in `config.user_mapping`, but, unfortunately, I still got the same results.

**Comment by vim-zz** [comment]
@yakirm-cr this is by design, the idea was that when using `rankByGitBlame ` to assign reviewers automatically with `add-reviewers@v1` then mapping users to `null` is a way to prevent the automatic mapping in certain cases, like in your example contributors that are not longer part of the team. 

```
- action: add-reviewers@v1
  args:
    reviewers: {{ repo | rankByGitBlame(gt=25) }}  <--- this will drop `null` users
```

On the other hand, when using `explainRankByGitBlame` with `add-comment@v1` it still shows these users details in the PR comment suggestion as this info might be valuable by itself. 

tl;dr the comment will not suggest `null` users but will show them in the details.

**Comment by yakirm-cr** [comment]
@vim-zz Thanks for the clarification!
I suggest to add it to the documentation (I suspect other may face the same and raise it again).

**Comment by vim-zz** [comment]
Added - thank you for the feedback!
